apps/spruce/src/components/PatchesPage/usePatchesQueryParams.ts

```diff
@@ -13,7 +13,7 @@ import { PatchPageQueryParams, ALL_PATCH_STATUS } from "types/patch";
  */
 export const usePatchesQueryParams = (): Omit<
   Required<PatchesInput>,
-  "includeCommitQueue" | "onlyCommitQueue"
+  "includeCommitQueue" | "onlyCommitQueue" | "requesters"
```

Comment:
This change was added because all of the params in `PatchesInput` are required and `requesters` was added in https://github.com/evergreen-ci/evergreen/commit/7c2c9f4fdbf6709fa914e2366fc3163b6552460e it should be updated in https://github.com/evergreen-ci/ui/pull/130

@SupaJoon  just wanted to flag with you for visibility

Reply:
FYI I merged in this same change in https://github.com/evergreen-ci/ui/pull/109 to fix the type errors, so Arjun feel free to undo in your PR!